### ISSUE TITLE
PDF file not Found

### ISSUE THREAD

**Issue by masifpak** [spender-sandbox/cuckoo-modified#512]
Dear Folks, I have enable reportpdf in cuckoo-modified but when I click on the following click it prompt "File not Found" and there is nor pdf file in directory too.

Could be it due to "Error: Analysis failed: invalid string pointer 0x03B228B000000000"

**Comment by doomedraven** [comment]
yes, and stop using dead project and move to alive  https://github.com/ctxis/cape
